Library/Homebrew/cmd/sh.rb

```diff
@@ -13,7 +13,7 @@ def sh
     ENV.activate_extensions!
 
     if superenv?
-      ENV.x11 = MacOS::X11.installed?
+      ENV.x11_if_installed
```

Comment:
I find this massively confusing, also when compared to what `x11` means in `stdenv`. But oh well, I guess legacy …